elements/bulbs-slick-slideshow/slick.scss

```diff
@@ -0,0 +1,98 @@
+/* Slider */
+
+.slick-slider {
+    position: relative;
+    display: block;
+    box-sizing: border-box;
+    -webkit-touch-callout: none;
+    -webkit-user-select: none;
```

Comment:
Ah yeah, I see a lot more is going on here than just prefixes. We should make a mixin and also reduce the number of prefixes if we don't need them.